src/SignalR/perf/benchmarkapps/Crankier/Readme.md

```diff
@@ -0,0 +1,40 @@
+# Crankier
+
+Load testing for ASP.NET Core SignalR
+
+## Commands
+
+### local
+
+```
+> dotnet run help local
+
+Usage:  local [options]
+
+Options:
+  --target-url <TARGET_URL>                   The URL to run the test against.
+  --workers <WORKER_COUNT>                    The number of workers to use.
+  --connections <CONNECTION_COUNT>            The number of connections per worker to use.
+  --send-duration <SEND_DURATION_IN_SECONDS>  The send duration to use.
+  --transport <TRANSPORT>                     The transport to use (defaults to WebSockets).
+  --worker-debug                              Provide this switch to have the worker wait for the debugger.
+```
+
+Notes:
+
+* `TARGET_URL` needs to be the route to your hub exposed by `UseSignalr` in your application.
+* `CONNECTION_COUNT` defaults to 10,000.
```

Comment:
Might as well add the defaults for `SEND_DURATION_IN_SECONDS` (which is `300`) and `WORKER_COUNT` (which is `1`)

Reply:
done